Pester.psm1

```diff
@@ -1515,6 +1516,15 @@ function Assert-VerifiableMocks {
 
 }
 
+function Set-ConflictingParameterNames {
```

Comment:
@FISHMANPET Could you move the function back to the Mock.ps1 file. The less we can pollute the Pester.psm1 file the better